packages/shared/styles/variables/_typography.scss

```diff
@@ -1,36 +1,38 @@
 @import "../helpers";
 :root {
   // font-family
-  --font-family-primary: "Roboto", serif;
-  --font-family-secondary: "Raleway", sans-serif;
+  --font-family--primary: "Roboto", serif;
+  --font-family--secondary: "Raleway", sans-serif;
   // font-weight
   --font-light: 300;
```

Comment:
If the variables responsible for `font-size` are called` --font-size - size`, then maybe the variables responsible for `font-weight` should be written as` --font-weight - weight`, wdyt?